api/src/main/java/org/openmrs/module/appointmentscheduling/api/impl/AppointmentServiceImpl.java

```diff
@@ -295,7 +295,9 @@ public AppointmentBlock voidAppointmentBlock(
 			Context.getService(AppointmentService.class).voidTimeSlot(timeSlot,
 					reason);
 		}
-
+                //Set Appointment location to Unknown Location
+                Location location=new Location(1);
```

Comment:
Then the only option we have is to set location to null. But for that I have to set location column a nullable. Is that okay?